openedx/features/qverse_features/registration/signals.py

```diff
@@ -74,6 +77,8 @@ def create_users_from_csv_file(sender, instance, created, **kwargs):
                          .format(instance.admission_file.path, err))
     csv_file.close()
     _write_status_on_csv_file(instance.admission_file.path, output_file_rows)
+    new_students = [student for student in output_file_rows if student.get('status') == 'Created']
```

Comment:
Lets create some constants for these statuses.

Reply:
done.